ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/AwsSecurityHubIntegrationForm.tsx

```diff
@@ -23,6 +24,7 @@ export type AwsSecurityHubIntegration = {
         credentials: {
             accessKeyId: string;
             secretAccessKey: string;
+            stsEnabled: boolean;
```

Comment:
Looking forward to day when we remove this redundant type declaration, can you also in frontend file which corresponds to backend source of truth:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/types/notifier.proto.ts#L43-L46